src/client/components/ModalViewCancelEvent/ModalViewCancelEvent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Modal from '../Modal/Modal';
+import { Button } from '../Button/Button';
+import PropTypes from 'prop-types';
+
+export const ModalCancelEvent = ({ cancelMessage }) => {
+  return (
+    <div>
+      <Modal title="Cancel Event">
+        <p>Are you sure you want to cancel</p>
+        <Button
+          className="modal-cancel-event-button"
+          appearance="danger"
+          onClick={cancelMessage}
+        >
+          Cancel
+        </Button>
+      </Modal>
+    </div>
+  );
+};
+ModalCancelEvent.propTypes = {
+  cancelMessage: PropTypes.func,
+};
+
+ModalCancelEvent.defaultProps = { cancelMessage: '' };
```

Comment:
The default value for function cannot be a string - that will cause an error. Instead, pass `null`.

Reply:
passed null